src/dotnet/APIView/APIView/HTMLRendererAPIV.cs

```diff
@@ -7,14 +7,14 @@ public class HTMLRendererAPIV : TreeRendererAPIV
     {
         protected override void RenderClassDefinition(StringBuilder builder, NamedTypeAPIV nt)
         {
-            builder.Append("<span id=\"").Append(EscapeHTML(nt.NavigationID)).Append("\" class=\"class\">")
-                .Append(EscapeHTML(nt.Name)).Append("</span>");
+            builder.Append("<a href=\"#commentThread\" id=\"").Append(EscapeHTML(nt.NavigationID)).Append("\" class=\"class commentable\">").
+                Append(EscapeHTML(nt.Name)).Append("</a>");
         }
 
         protected override void RenderEnumDefinition(StringBuilder builder, NamedTypeAPIV nt)
         {
-            builder.Append("<span id=\"").Append(nt.NavigationID).Append("\" class=\"enum\">")
-                .Append(EscapeHTML(nt.Name)).Append("</span>");
+            builder.Append("<a href=\"#commentThread\" id=\"").Append(nt.NavigationID).Append("\" class=\"enum commentable\">").
```

Comment:
You can remove href=\"#commentThread\" and wire the same behavior in JS.

Reply:
You would need to do it in JS for the next steps anyway .

Reply:
Good point - was going to implement that but forgot. On it right now